dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/JdbcValidationResultTableManager.java

```diff
@@ -113,7 +138,7 @@ protected void populateTable( AnalyticsTableUpdateParams params, AnalyticsTableP
     {
         final String tableName = partition.getTempTableName();
 
-        String insert = "insert into " + partition.getTempTableName() + " (";
+        StringBuilder insert = new StringBuilder("insert into " + partition.getTempTableName() + " (");
```

Comment:
Could we revert this change - Java is converting to String builder internally anyway.